helm/loki/templates/query-frontend/service-query-frontend.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
 spec:
   type: ClusterIP
   clusterIP: None
+  publishNotReadyAddresses: true
```

Comment:
Added this because one of the query-frontend pods logged `number of queriers connected to query-frontend is 0`

A quick search resulted in this https://github.com/cortexproject/cortex/discussions/3866#discussioncomment-429966 suggesting to add `publishNotReadyAddresses` but on an additional headless service.

@webwurst do you have thoughts on this?

Reply:
This is interesting. Need to read up a bit about that.

Reply:
Tried a clean install (deleted the old app + deleted loki ns) and this seems to work out of the box. Question is if this is actually bad practice

Reply:
Before and after this change seems to be the same. Before, the querier-frontend did not have a readiness or liveness probe. In my head the end effect is the same. If a replica is down, the request fails